features/repost-activity.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Reposting an object
```

Comment:
Maybe we use the terminology `post` instead of `object` in this?

Reply:
Updated in 9faffb4b1a509c7027d792aba0b493008496e9c9